deal.II-toolchain/packages/dealii.package

```diff
@@ -136,10 +135,8 @@ fi
 
 if [ ! -z "${P4EST_DIR}" ]; then
     cecho ${INFO} "deal.II: configuration with P4EST"
-    CONFOPTS="\
-        ${CONFOPTS} \
-        -D DEAL_II_WITH_P4EST:BOOL=ON \
-        -D P4EST_DIR=${P4EST_DIR}"
```

Comment:
don't we need this anymore?

Reply:
Precisely: Line 142 `-D P4EST_DIR=${P4EST_DIR}"`

Reply:
Oh yeah, this is not needed any more. Thanks!

Reply:
@koecher sorry, you are absolutely right, this was deleted by a too fast rebase... let me add this again.... (I was at an other line where I exactly removed such a case)...